CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### 💡 Enhancements 💡
+
+- `tailsamplingprocessor`: Add support for all sampling policies in and & composite policy (#11505)
+
+### 🧰 Bug fixes 🧰
+
+- `tailsamplingprocessor`: Fix runtime crash caused by invalid composite policy configuration (#11505)
```

Comment:
I trust you fixed this, but can you point me to the test that actually asserts this?

Reply:
The cause was that both the xxx_helper.go implementation did not return any errors previously. That should now be asserted by the two new `unsupported sampling policy type` test cases.